examples/image_warping_example.py

```diff
@@ -0,0 +1,83 @@
+"""
+Mouse event callbacks
+=====================
+
+Warp an image based on points moved by user using mouse event callbacks in
+napari.
+
```

Comment:
```suggestion
Derived from scikit-image example: https://scikit-image.org/docs/stable/auto_examples/transform/plot_tps_deformation.html
```
Not sure how best to add this, but I think it's worth it

Reply:
I think this is a good suggestion!